client/src/pages/customer-pages/Home/home.css

```diff
@@ -0,0 +1,21 @@
+@media screen and (max-width: 500px) {
```

Comment:
Sorry to spam you with all these :) Here's another spot where some of the styles probably apply to all browser widths, and some others just to max-width: 500px